scripts/lighthouseConfig.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    extends: 'lighthouse:default',
+    settings: {
+      skipAudits: [
+        'is-on-https',
+      ],
+    },
+  };
```

Comment:
Would this mean that this audit is also skipped on test as well as locally?

Reply:
Sorry just seen https://github.com/bbc/simorgh/pull/2044#issuecomment-505454752

I think it's low risk but I think it would be good to have an issue to get this audit to be skipped locally but still run in production, please could you create one?

Reply:
This config needs manually loaded using the `--config-path` command line argument.  I've done that in the package.json here for the two lighthouse scripts that run tests against localhost.bbc.com

Test is handled over in the infrastructure repo it seems?

https://github.com/bbc/simorgh-infrastructure/blob/f856734faa829edb06e82afa214443ed5af9421f/scripts/run_lighthouse.sh

This file isn't being used there, so the https audit should still run

Reply:
Great got it - thanks!